### ISSUE TITLE
PVD Thoughts (Did not deliver)

### ISSUE THREAD

**Issue by nayana487** [ga-wdi-boston/js-functions-ins-and-outs#27]
I didn't teach this lesson at all. I felt that what was covered here had already been/will be covered in past/future lessons. I think the only thing that isn't/won't be covered in other lessons is the `arguments` object but I didn't think that was pertinent enough to warrant having to teach the students in person. I actually didn't know about that myself, so super cool to learn! 

That said, we did give this lesson to students and the study as well. We told them to go through these on their own time and informed them that they will have a diagnostic on this Monday morning. I personally don't think they should have a diagnostic on this, but my team disagreed and I was OK with that. 

Instead, I delivered this lesson on CSS: https://github.com/ga-wdi-lessons/css-review. It went really well. They were especially impressed with how they can change HTML and CSS with the inspector. I think this is going to set them up well for the DOM lesson. 

I also had them do this lesson: https://github.com/ga-wdi-lessons/html-intro on their own prior to the CSS class. 

As a preview of next week, we plan to cover layout, DOM and jquery, events and callbacks, scope and context, and es6 features. The plan is to get into APIs and AJAX right after Thanksgiving, but before they start project 1. We will make sure that the team that remains is set up for success on delivering that.

**Comment by jrhorn424** [comment]
@nayana487 Thanks so much for your feedback. It was thoughtful and there are some action items I can take from this.

Particularly useful were the links to the CSS and HTML resources, since we've received feedback that our HTML and CSS talks could use some beefing up.

I'll add an issue to the appropriate repos in our org. :smile:

**Comment by jrhorn424** [comment]
@nayana487 I've opened a few issues over at `html-css*` (noted above). Is this closable, or did I miss anything addressable in your OP?

The OP communicated the following clearly:

- Your opinion is that this material was duplicative of earlier lessons.
- Your opinion is that `arguments` wasn't pertinent to the goals of `js` sequence.

Is this accurate?

**Comment by nayana487** [comment]
@jrhorn424 Yes. Also, you can close all the issues I posted. I was just documenting where I diverted in PVD delivery but those comments might no longer be relevant.
